.ci/install_qemu.sh

```diff
@@ -54,7 +54,7 @@ uncompress_static_qemu() {
 
 build_and_install_static_qemu() {
 	build_static_qemu
-	uncompress_static_qemu "${KATA_TESTS_CACHEDIR}/${QEMU_TAR}"
+	uncompress_static_qemu "${QEMU_TAR}"
```

Comment:
I am not sure this will work. The `build_static_qemu` is moving the `$QEMU_TAR` to `$KATA_TESTS_CACHEDIR`, see: https://github.com/kata-containers/tests/blob/bbed59bce242a7d24cd9f04d5fa87fdab788c4cf/.ci/install_qemu.sh#L46

so `uncompress_static_qemu` will not found the `$QEMU_TAR` in the current working directory